lerna.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
+  "version": "independent",
+  "npmClient": "yarn",
+  "verbose": true,
+  "packages": [
+    "./packages/*"
+  ],
+  "conventionalCommits": true,
+  "changelog": true,
+  "message": "chore(release): publish",
+  "command": {
+    "publish": {
+      "tag": true,
+      "push": true,
+      "pushArgs": [
+        "--follow-tags"
+      ],
+      "yes": true,
+      "registry": "http://192.168.1.169:4873/"
```

Comment:
Should we update this before merging it?

Reply:
I was thinking about it.. but maybe removing together with new workflow would be better.